wappsto/connection/communication.py

```diff
@@ -486,6 +484,9 @@ def send_data(self, data):
                 self.get_object_without_none_values(data_element)
                 if len(data_element) == 0:
                     data.remove(data_element)
+                else:
+                    if data_element.get("method", "") == "PUT":
```

Comment:
Why is it only PUT method?

Reply:
it was previously done only for put

Reply:
But if you send a DELETE/POST/GET is it then just not handled?